tests/test_tasks.py

```diff
@@ -1128,4 +1128,4 @@ async def test_telemetry_uptake_tasks_exists(self):
         received = await telemetry.main_summary_uptake('firefox', '57.0a1')
         assert received["status"] == Status.EXISTS.value
         assert received["message"] == ("Telemetry uptake for version 57.0a1 "
-                                       "({}) is 55.0%".format(build_id))
+                                       "({}) is 55.000%".format(build_id))
```

Comment:
Nit: Looks like the expected value is: 

```suggestion
                                       "({}) is 55.001%".format(build_id))
```


```
E       AssertionError: assert 'Telemetry up...6) is 55.001%' == 'Telemetry up...6) is 55.000%'
E         
E         Skipping 51 identical leading characters in diff, use -v to show
E         - ) is 55.000%
E         ?           ^
E         + ) is 55.001%
E         ?           ^

```

https://app.circleci.com/pipelines/github/mozilla/PollBot/632/workflows/348fdc5b-4415-488e-8904-231ec913bfd7/jobs/1226?invite=true#step-106-20858_24